Objects/bytesobject.c

```diff
@@ -1184,25 +1184,28 @@ PyObject *PyBytes_DecodeEscape(const char *s,
         unsigned char c = *first_invalid_escape;
         if ('4' <= c && c <= '7') {
             if (PyErr_WarnFormat(PyExc_DeprecationWarning, 1,
-                                 "invalid octal escape sequence '\\%.3s'",
-                                 first_invalid_escape) < 0)
+                                 "'\\%.3s' is an invalid escape sequence. "
+                                 "Such sequences will not work in the future. "
+                                 "Did you mean \"\\\\%.3s\"? A raw string is also an option.",
+                                 first_invalid_escape, first_invalid_escape) < 0)
             {
                 Py_DECREF(result);
                 return NULL;
             }
         }
         else {
             if (PyErr_WarnFormat(PyExc_DeprecationWarning, 1,
-                                 "invalid escape sequence '\\%c'",
-                                 c) < 0)
+                                 "'\\%c' is an invalid escape sequence. "
```

Comment:
Is there precedent for error messages or other documentation referring to `bytes` objects as "strings" like this? The wording "raw string" seems like a bad way to describe `rb''` if the goal is to encourage clear thinking about the Unicode/bytes dichotomy.

Reply:
Even if I modify this specific message, it does not seem to change the warning message that is raised for me when I run `b'\\w'` but regardless raw byte strings are called raw byte strings, they're raw strings but byte strings. Adding a dynamic message could be difficult.

Reply:
Resolved by dropping suggestion in this case.